internal/build/docker.go

```diff
@@ -72,18 +73,19 @@ func (l *localDockerBuilder) buildDocker(ctx context.Context, df Dockerfile,
 		return "", fmt.Errorf("buildFromDfWithFiles: %v", err)
 	}
 
-	newDigest, err := l.execStepsOnImage(ctx, baseDigest, steps)
-	if err != nil {
-		return "", fmt.Errorf("execStepsOnImage: %v", err)
-	}
+	cID, err := l.pool.claimContainer(ctx, baseDigest)
```

Comment:
maybe docstring: "start a container from base image so we can exec steps on it" or something

Reply:
done